Domains/4-Application/DrawingProduction/DrawingProductionExperimental.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please fix the fix the file extension. It should be **.ecschema.xml**.

`DrawingProductionExperimental.xml` -> `DrawingProductionExperimental.ecschema.xml`